bundles/org.eclipse.swt/Eclipse SWT/win32/org/eclipse/swt/graphics/Region.java

```diff
@@ -47,6 +49,8 @@ public final class Region extends Resource {
 	 */
 	public long handle;
 
+	private int initalZoom;
```

Comment:
```suggestion
	private int initialZoom;
```

Reply:
Yeah it needed some cleaning but its all fixed now